### ISSUE TITLE
Add version tags to the repository

### ISSUE THREAD

**Issue by JustAnotherArchivist** [webrecorder/warcio#126]
While trying to figure out which version of warcio first included some functionality (for specifying a minimum dependency version), I realised that this repository unfortunately has no version tags. It's therefore not easily possible to determine which version first contained a particular commit, which is usually done with `git tag --contains COMMITID` (and also displayed on the web interface on most Git hosting platforms). It also appears that the version bump in `setup.py` does not always coincide with the actual release, so that's not a reliable way to check either.

Would you consider retroactively adding tags for the releases, ideally back to version 1.0?

**Comment by wumpus** [comment]
I am for it,  but it would require some sleuthing. Have you already made a list of the ids? Looks like it would require a bit of looking around, perhaps with a good ole' `git log -p . > everything-ever.txt`

**Comment by JustAnotherArchivist** [comment]
Yeah, it definitely would. Possibly some comparisons to the tars on PyPI as well.
I haven't done anything regarding this so far.
